tools/example.py

```diff
@@ -178,24 +201,10 @@ async def main() -> None:
     await coap_context.initialize(args.coap_port, args.listen_ip_address)
     await ws_context.initialize(args.ws_port, args.ws_api_url)
 
-    if not args.init and not (args.gen1 or args.gen2 or args.gen3 or args.gen4):
+    if not args.init and not args.gen:
```

Comment:
We use the gen1, gen2, ... in other places that needs to be fixed also:
https://github.com/home-assistant-libs/aioshelly/blob/377414fa8c4bba4c3753b50ce63367bc18f33e36/tools/example.py#L212-L214